PhysicsTools/Heppy/python/physicsobjects/Electron.py

```diff
@@ -59,15 +63,22 @@ def mva_passes(self, name, wp):
             if id in miniAODids :
                 passed = self.electronIDs()[miniAODids.index(id)][1]
             elif id in wps_dict.keys() :
-                if name not in self._mvaid_score or name not in self._mvaid_category :
-                    score_raw, category = self._mva_score_and_category(name)
                 FWLitename, FWLitewp = wps_dict[id]
-                passed = working_points[FWLitename].passed(
-                    self.physObj,
-                    self._mvaid_score[name],
-                    self._mvaid_category[name],
-                    FWLitewp
-                    )
+                if 'cutBased' in name :
```

Comment:
we should probably change the name of this method (and its docstring) to id_passes